src/FluentMigrator.Abstractions/Builders/IColumnTypeSyntax.cs

```diff
@@ -134,7 +136,7 @@ public interface IColumnTypeSyntax<TNext> : IFluentSyntax
         /// <param name="size">The number of digits</param>
         /// <param name="precision">The number of digits after the comma</param>
         /// <returns>The next step</returns>
-        TNext AsDecimal(int size, int precision);
+        TNext AsDecimal(int size, [ValueRange(0, 28)] int precision);
```

Comment:
Please add a <remarks><see cref="System.Decimal" /> type in C# supports 28 digits of precision, and the default behavior of FluentMigrator is to match that.</remarks>